### PR TITLE
v2.9.x: Change CI and nightlies to use Go 1.20

### PR DESCRIPTION
**Pull request by wallyqs**
Includes fix for test `TestGatewayTLSMixedIPAndDNS ` that was breaking in Go 1.20.

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -6,7 +6,9 @@ vm:
 
 language: go
 go:
+  - "1.20.x"
   - "1.19.12"
```
Comment (derekcollison) [inline .travis.yml:10]: Do we want to double our builds here?

**Review by derekcollison — CHANGES_REQUESTED** [review]
I would replace with Go 1.20, since 1.19 deprecated would not run 2 builds.
Otherwise LGTM.

File: .travis.yml (hunk 1 above)
Comment (wallyqs) [reply .travis.yml]: Updated to removed v1.19 from the build

**Review by derekcollison — APPROVED** [review]
LGTM

**Pull request merged**